src/internal/clusterstate/v2.8.0/collection.go

```diff
@@ -239,3 +239,66 @@ func migratePostgreSQLCollection(ctx context.Context, tx *pachsql.Tx, name strin
 	}
 	return nil
 }
+
+func setupPostgresCollections(ctx context.Context, sqlTx *pachsql.Tx, collections ...*postgresCollection) error {
+	for _, col := range collections {
+		columns := []string{
+			"createdat timestamp with time zone default current_timestamp",
+			"updatedat timestamp with time zone default current_timestamp",
+			"proto bytea",
+			"version text",
+			"key text primary key",
+		}
+
+		indexFields := []string{"'key'"}
```

Comment:
I don't think we need to specify an index on a primary key. They should already be indexed